### ISSUE TITLE
Problems with LC_ALL and check_ping, check_load... [sf#1266157]

### ISSUE THREAD

**Issue by monitoring-user** [monitoring-plugins/monitoring-plugins#473]
Submitted by alien2108 on 2005-08-22 14:59:37

With the latest nagios_plugins
(nagios-plugins-1.4-2.rf) compiled by myself I have
problems with numerious check scripts.

For example if I run check_ping:
# ./check_ping1 -H 192.168.10.2 -w10,20% -c

50,50%/bin/ping -n -U -w 10 -c 5 192.168.10.2
CRITICAL - Could not interpret output from ping command

BUT:

LC_ALL=C ./check_ping1 -H 192.168.10.2 -w10,20% -c 50,50%
PING OK - Packet loss = 0%, RTA = 0.23 ms

Is this problem with plugins not correctly handling
locales? System is Linux Mandrake 10.0 with LC's:

LC_ADDRESS=sl_SI
LC_COLLATE=en_US
LC_CTYPE=en_US
LC_IDENTIFICATION=sl_SI
LC_MEASUREMENT=sl_SI
LC_MESSAGES=en_US
LC_MONETARY=sl_SI
LC_NAME=sl_SI
LC_NUMERIC=sl_SI
LC_PAPER=sl_SI
LC_SOURCED=1
LC_TELEPHONE=sl_SI
LC_TIME=en_US
MAILCHECK=60
